### PR TITLE
add Shenzhen Technology University and University Kebangsaan Malaya

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request is a fixup of #14 ny @StevE-Ong 

@StevE-Ong please check if I did not destroy your code. You should still be main  author.

### CODE REVIEW

File: data/picongpu-community.geojson [hunk 1]
```diff
@@ -481,6 +499,24 @@
           48.7142971
```
Comment (PrometheusPi) [inline data/picongpu-community.geojson:506]: This might not be correct? Shall I remove this part? (Where does it come from?

Comment (StevE-Ong) [reply data/picongpu-community.geojson:506]: I forgot where it comes from, I think it should be removed.

**Comment by StevE-Ong** [comment]
Thanks @PrometheusPi. Sorry for the troubles. I think it is OK without the section `48.714..`

File: data/picongpu-community.geojson (hunk 1 above)
Comment (StevE-Ong) [reply data/picongpu-community.geojson:506]: I think it was not from me. It was already [there.](https://github.com/ComputationalRadiationPhysics/picongpu-communitymap/blob/b946c30b95b2485feeea959d6f0d7412e5dcbef0/data/picongpu-community.geojson?short_path=da71226#L499)

**Comment by PrometheusPi** [comment]
@StevE-Ong Thanks for you review - I will fix it.

**Comment by sbastrakov** [comment]
@PrometheusPi i think we forgot about this one, should it just be merged (if so - i have no rights in this repo)?

**Comment by PrometheusPi** [comment]
I found out that the unknown institute was added by @ax3l in 2015 https://github.com/ComputationalRadiationPhysics/picongpu-communitymap/commit/92d8538f24e7b3ad8cbd9c0e8d2b700c8b39532d I added the institute with name

**Review by steindev — APPROVED** [review]

**Pull request merged**